### PR TITLE
Vendor packages for NVIDIA H100 GPU

### PR DESCRIPTION
**Pull request by JamesLiang29**

### CODE REVIEW

**Comment by zawachte** [comment]
is this needed with the gpucount pr going in?

**Comment by JamesLiang29** [comment]
> is this needed with the gpucount pr going in?

It's plan B in case GPU count PR not fully functioning. Moreover, even with GPU count PRs in, we need several large scale VM SKUs.

**Comment by zawachte** [comment]
But shouldn’t the moc tag be bumped either way?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by JamesLiang29** [comment]
> But shouldn’t the moc tag be bumped either way?

Yes, depends on the merge order one of us may be covered by another and discarded.

File: go.mod [hunk 1]
```diff
@@ -9,8 +9,8 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v1.4.1
 	github.com/golang/mock v1.6.0
-	github.com/microsoft/moc v0.18.1
-	github.com/microsoft/moc-sdk-for-go v0.18.1
+	github.com/microsoft/moc v0.19.1-shengl-h100
+	github.com/microsoft/moc-sdk-for-go v0.19.0-shengl-h100
```
Comment (JamesLiang29) [inline go.mod:13]: Depends on https://github.com/microsoft/moc/pull/265

**Comment by zawachte** [comment]
> > But shouldn’t the moc tag be bumped either way?
> 
> Yes, depends on the merge order one of us may be covered by another and discarded.

But the other PR already merged

**Comment by JamesLiang29** [comment]
> But the other PR already merged

This one is newer, contains changes for NVIDIA H100 support.

**Review by zawachte — APPROVED** [review]

**Pull request merged**